web/src/pages/howitworks/howitworksrework.css

```
@@ -1,297 +1,99 @@
-.howitworks__main {
-  width: 100%;
-  margin-left: auto;
-  margin-right: auto;
-  padding-left: 5rem;
-  padding-right: 5rem;
-  display: flex;
-  justify-content: center;
-  box-sizing: border-box;
-  flex-direction: column;
-}
-
-.span{
-  font-weight: bold;
+/* Algemene stijlen */
```

Global styling mag absoluut niet

```suggestion
/* Algemene stijlen */
body {
  font-family: "Poppins", sans-serif;
  color: #333;
  background-color: #f8f9fa;
  margin: 0;
  padding: 0;
}
```